easybuild/easyconfigs/g/gompi/gompi-2015b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '2015b'
+
+homepage = '(none)'
```

Comment:
I was half-kidding, we could use a docs page explaining what toolchains are, which ones we have (with versions included), etc.